python/lsst/meas/astrom/astromLib.i

```diff
@@ -11,26 +11,61 @@ Python interface to lsst::meas::astrom
 %{
 #include "lsst/afw/image.h"
 #include "lsst/afw/cameraGeom.h"
+#include "lsst/afw/geom.h"
 #include "lsst/afw/table.h"
 #include "lsst/afw/image/Wcs.h"
 #include "lsst/afw/image/TanWcs.h"
 #include "lsst/meas/astrom/matchOptimisticB.h"
 #include "lsst/meas/astrom/makeMatchStatistics.h"
+#include "lsst/meas/astrom/PolynomialTransform.h"
+#include "lsst/meas/astrom/SipTransform.h"
+#include "lsst/meas/astrom/ScaledPolynomialTransformFitter.h"
 %}
 
 %include "lsst/p_lsstSwig.i"
 %initializeNumPy(meas_astrom)
-
+%{
+#include "ndarray/converter.h"
+#include "ndarray/converter/eigen.h"
+%}
+%include "ndarray.i"
 %include "lsst/pex/config.h"
 
 %shared_ptr(lsst::meas::astrom::MatchOptimisticBControl);
 
 %import "lsst/afw/table/tableLib.i"
 %import "lsst/afw/image/wcs.i"
 %import "lsst/afw/math/statistics.i"
+%import "lsst/afw/geom/geomLib.i"
 
 %include "lsst/meas/astrom/matchOptimisticB.h"
 
+%include "std_pair.i"
+
+namespace std { // have to do this in namespace block so Swig can find size_t
+%template(DoubleSizePair) pair<double,size_t>;
+}
+
+%declareNumPyConverters(ndarray::Array<double const,2,2>)
```

Comment:
I was actually planning to put this in the custom .i file for the transform classes, since it's only needed by them.